ocpmodels/datasets/ase_datasets.py

```diff
@@ -102,15 +102,12 @@ def __getitem__(self, idx):
                 atoms, **self.config.get("atoms_transform_args", {})
             )
 
-        if "sid" in atoms.info:
-            sid = atoms.info["sid"]
-        else:
-            sid = tensor([idx])
+        sid = atoms.info.get("sid", str(self.ids[idx]))
+        fid = atoms.info.get("fid", tensor([0]))
 
         # Convert to data object
         data_object = self.a2g.convert(atoms, sid)
-
-        data_object.pbc = tensor(atoms.pbc)
```

Comment:
What's the rationale for deleting this? `radius_graph_pbc` will always assume True,True,True otherwise.